virtualization/windowscontainers/management/container_networking.md

```diff
@@ -235,13 +266,58 @@ C:\> docker run -it --network=MyTransparentNet --ip=10.80.123.32 windowsserverco
 
 Static IP assignment is performed directly on the container's network adapter and must only be performed when the container is in a STOPPED state. "Hot-add" of container network adapters or changes to the network stack is not supported while the container is running.
 
+## Docker Compose and Service Discovery
+
+> For a practical example of how Docker Compose and Service Discovery can be used to define multi-service, scaled-out applications, visit [this post](https://blogs.technet.microsoft.com/virtualization/2016/10/18/use-docker-compose-and-service-discovery-on-windows-to-scale-out-your-multi-service-container-application/) on our [Virtualization Blog](https://blogs.technet.microsoft.com/virtualization/).
+
+### Docker Compose
+
+[Docker Compose](https://docs.docker.com/compose/overview/) can be used to define and configure container networks alongside the containers/services that will be using those networks. The Compose 'networks' key is used as the top-level key in defining the networks to which containers will be connected. For example, the syntax below defines the default NAT network created by Docker to be the 'default' network for all containers/services defined in a given Compose file.
+
+```none
+networks:
+ default:
+  external:
+   name: "nat"
+```
+For further information on defining/configuring container networks using Docker Compose, refer to the [Compose File reference](https://docs.docker.com/compose/compose-file/).
```

Comment:
Can you add a second example for defining a user-defined nat network which is a partition of the pre-existing NAT internal prefix (I think we included this in the blog article).